CommandLine/emake/Makefile

```diff
@@ -43,8 +43,8 @@ DEPENDS := $(OBJECTS:.o=.d)
 
 CXXFLAGS += -I$(SHARED_SRC_DIR) $(addprefix -I$(SHARED_SRC_DIR)/, $(SHARED_INCLUDES))
 SOURCES += $(addprefix $(SHARED_SRC_DIR),$(SHARED_SOURCES))
-OBJECTS += $(addprefix $(OBJ_DIR)/shared/,$(SHARED_SOURCES:.cpp=.o))
-DEPENDS += $(addprefix $(OBJ_DIR)/shared/,$(SHARED_SOURCES:.cpp=.d))
+OBJECTS += $(addprefix .eobjs/shared/,$(SHARED_SOURCES:.cpp=.o))
```

Comment:
Did you mean to make this change? Or was this what you were trying when I asked you to make substitutions? This isn't the right makefile for that. I think you can just revert this.